asserts/asserts.go

```diff
@@ -58,6 +58,7 @@ func (at *AssertionType) MaxSupportedFormat() int {
 var (
 	AccountType         = &AssertionType{"account", []string{"account-id"}, assembleAccount, 0}
 	AccountKeyType      = &AssertionType{"account-key", []string{"public-key-sha3-384"}, assembleAccountKey, 0}
+	RepairType          = &AssertionType{"repair", []string{"repair-id"}, assembleRepair, 0}
```

Comment:
I updated the branch now to have (brand-id, repair-id) as the primary key. even if we don't open this up to brands it should not do any harm and make things easier for the future.